services/ui-src/src/components/statusing/StatusTable.tsx

```diff
@@ -0,0 +1,130 @@
+import { Fragment, useContext } from "react";
+import { useNavigate } from "react-router-dom";
+// components
+import { Box, Button, Image, Td, Tr } from "@chakra-ui/react";
+import { ReportContext, Table } from "components";
+// types
+import { ReportPageProgress } from "types";
+// utils
+import { getRouteStatus } from "utils";
+// verbiage
+import verbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
+// assets
+import editIcon from "assets/icons/icon_edit.png";
+import errorIcon from "assets/icons/icon_error_circle_bright.png";
+
+export const StatusTable = () => {
+  const { report } = useContext(ReportContext);
+  const { review } = verbiage;
+  const rowDepth = 1;
+  return report ? (
+    <Box sx={sx.container}>
+      <Table content={review.table}>
+        {getRouteStatus(report).map((page: ReportPageProgress) => {
+          return <ChildRow key={page.path} page={page} depth={rowDepth} />;
+        })}
+      </Table>
+    </Box>
+  ) : (
+    <Box />
+  );
+};
+
+const ChildRow = ({ page, depth }: RowProps) => {
+  const { name, children } = page;
+  return (
+    <Fragment key={name}>
+      <TableRow page={page} depth={depth} />
+      {children?.map((child) => (
+        <ChildRow key={child.path} page={child} depth={depth + 1} />
+      ))}
+    </Fragment>
+  );
+};
+
+const TableRow = ({ page, depth }: RowProps) => {
+  const navigate = useNavigate();
+  const { name, path, children, status } = page;
+
+  return (
+    <Tr>
+      {depth == 1 ? (
+        <Td sx={sx.parent}>{name}</Td>
+      ) : (
+        <Td pl={`${1.25 * depth}rem`}>{name}</Td>
+      )}
+      <Td sx={sx.status}>
+        {!status && status !== undefined && (
+          <>
+            <Image src={errorIcon} alt="Error notification" />
+            Error
+          </>
```

Comment:
Nit: Don't need fragment here.

Reply:
If we don't have a fragment, where does `Error` go?

Reply:
Do need a fragment here since it's 2 lines, but swapping to a more defined Box and adding a Text around the Error text

Reply:
Yeah that's my bad -- I didn't even see the text there. 🤦‍♂️
Agree that Nick's solution is the right way to go. We should always be wrapping text in the Chakra `Text` component.